scripts/docker-eth

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+mkdir -p ~/.ethereum ~/.web3
+
+# check if sudo is necessary
+if ! id -nG $(whoami)|grep -qw "docker"; then SUDO='sudo'; else SUDO=''; fi
```

Comment:
Hey @chfast thanks for your comment!
The sudo check is only about issuing the `docker run ...` command as non root (if applicable). The ownership of the process is taken care of with the
```
--user $(id -u):$(id -g)
```
bit in the proposed script. So the process in the container will have the user/group id of your current user.